scenarios/remote_signals/README.md

```diff
@@ -0,0 +1,36 @@
+## remote_signals
+
+### Description
+
+This scenario tests the throughput of `remote_signals` operations.
+
+**warning** This is a TryCP-based scenario and needs to be run differently to other scenarios.
```

Comment:
The following will render a warning box on GitHub. Just a suggestion, I'm not bothered either way :slightly_smiling_face: 

```suggestion
> [!WARNING]
> This is a TryCP-based scenario and needs to be run differently to other scenarios.
```